app/api/schema/settings.py

```diff
@@ -196,3 +196,16 @@ class Meta:
     smtp_password = fields.Str(allow_none=True)
     smtp_port = fields.Integer(allow_none=True)
     smtp_encryption = fields.Str(allow_none=True)  # Can be tls, ssl, none
+
+    # Admin Invoice Details
+    admin_billing_contact_name = fields.Str(allow_none=True)
+    admin_billing_phone = fields.Str(allow_none=True)
+    admin_billing_email = fields.Email(allow_none=True)
+    admin_billing_country = fields.Str(allow_none=True)
+    admin_billing_tax_info = fields.Str(allow_none=True)
+    admin_company = fields.Str(allow_none=True)
+    admin_billing_company_address = fields.Str(allow_none=True)
+    admin_billing_company_city = fields.Str(allow_none=True)
+    admin_billing_zip = fields.Str(allow_none=True)
+    admin_billing_additional_info = fields.Str(allow_none=True)
+    admin_billing_company_comments = fields.Str(allow_none=True)
```

Comment:
admin_billing_additional_info

Reply:
Already done!

Reply:
@mrsaicharan1 https://github.com/fossasia/open-event-server/blob/85f7fa80f8fcc229ed16a7d8d9feaf40f6d34edb/app/api/schema/settings.py

it's not reflecting.

Reply:
@mrsaicharan1 It'll automatically show outdated once you have done the change

Reply:
@CosmicCoder96 This is the required change?` admin_billing_additional_info`

Reply:
https://github.com/fossasia/open-event-server/blob/9ed109f0f2849df0968a07adf98c3e483210a755/app/api/schema/settings.py
@CosmicCoder96 Please have a look here

Reply:
@mrsaicharan1 . https://github.com/fossasia/open-event-server/blob/9ed109f0f2849df0968a07adf98c3e483210a755/app/api/schema/settings.py#L211

You are repearting this field. `admin_billing_comapany_comments`  has to be renamed to `admin_billing_additional_info` a new field is not required. That was the originally requested change.